### PR TITLE
docs: add develop docs

### PR DESCRIPTION
**Pull request by sarahgm**

### CODE REVIEW

File: docs/content/pages/develop/scripts.mdx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+caption: List of all the scripts.
+---
+
+We use several scripts to make developing Marigold easier and more convenient.
+
+Run `$ pnpm <script name>` in your favorite terminal.
+
+## Developing
+
+### `dev`
```
Comment (sebald) [inline docs/content/pages/develop/scripts.mdx:12]: There is no dev script anymore 😭

File: docs/content/pages/develop/release.mdx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Release
+caption: How we release Marigold.
+---
+
+Releases are automated through changeset’s release action. In order to cut a new release go to Marigold’s open pull requests. If there are any changes to be published, there is a pull request called `“release: version packages”`.
+
+Inside the pull request you will find a list of all the changes and version bumps that will happen. After you review the pull request, approve it and then merge it. This will trigger the `Release` action which will [publish packages to npm](https://www.npmjs.com/search?q=%40marigold), send a notification to our Slack channel and deploy the documentation.
+
+## Deploy Docs manually
```
Comment (sebald) [inline docs/content/pages/develop/release.mdx:10]: We don't have any of this yet.

File: docs/content/pages/develop/scripts.mdx [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Run the [TypeScript](https://www.typescriptlang.org/) compiler to find any inconsistent or wrong types.
+
+## Build
+
+### `build`
+
+Build all the packages.
+
+### `watch`
```
Comment (sebald) [inline docs/content/pages/develop/scripts.mdx:46]: This is now a dev command

**Review by sebald — CHANGES_REQUESTED** [review]

File: docs/content/pages/develop/release.mdx (hunk 2 above)
Comment (sarahgm) [reply docs/content/pages/develop/release.mdx:10]: what do you mean?

Comment (sebald) [reply docs/content/pages/develop/release.mdx:10]: with the new hosting on vercel we don't have a realy deployment right? :-O

File: docs/content/pages/develop/scripts.mdx (hunk 3 above)
Comment (sebald) [reply docs/content/pages/develop/scripts.mdx:58]: I think we just deleted it because we didn't use it anyway

**Review by sebald — CHANGES_REQUESTED** [review]

File: docs/content/pages/develop/release.mdx (hunk 2 above)
Comment (sarahgm) [reply docs/content/pages/develop/release.mdx:10]: ahhh hmm yes than we should rewrite it?

Comment (sarahgm) [inline docs/content/pages/develop/release.mdx:10]: @sebald what do you think?

File: docs/content/pages/develop/release.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Release
+caption: How we release Marigold.
+---
+
+Releases are automated through changeset’s release action. In order to cut a new release go to Marigold’s open pull requests. If there are any changes to be published, there is a pull request called `“release: version packages”`.
+
+Inside the pull request you will find a list of all the changes and version bumps that will happen. After you review the pull request, approve it and then merge it. This will trigger the `Release` action which will [publish packages to npm](https://www.npmjs.com/search?q=%40marigold), send a notification to our Slack channel and deploy the documentation.
+
+## Deploying with Vercel
```
Comment (sarahgm) [inline docs/content/pages/develop/release.mdx:10]: @sebald  what do you think about this?

**Review by sebald — APPROVED** [review]

**Pull request merged**